src/locale/locales/zh-TW/messages.po

```diff
@@ -601,7 +601,7 @@ msgstr "來自 <0/>"
 
 #: src/screens/Signup/StepInfo/Policies.tsx:74
 msgid "By creating an account you agree to the {els}."
-msgstr ""
+msgstr 建立帳戶即表示你同意 {els}。""
```

Comment:
I realise the PR isn't finished, I just wanted to point out one tiny thing here: one of the quotation marks is in the wrong place.

Reply:
Thanks for pointing this out, I'll fix it